backend/curriculum_tracking/management/commands/import_curriculum.py

```diff
@@ -0,0 +1,74 @@
+from core.models import Curriculum
+from curriculum_tracking.models import (
+    CurriculumContentRequirement,
+    ContentItemOrder,
+    ContentItem,
+)
+from curriculum_tracking.card_generation_helpers import get_ordered_content_items
+import json
+from django.core.management.base import BaseCommand
+import pandas as pd
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        json_file = "dev_helpers/data/intro-to-tilde-course.json"
+        save_curriculum_to_db(json_file)
+
+
+def save_curriculum_to_db(json_file):
+    with open(json_file) as f:
+        data = json.load(f)
+
+    df_dict = {i: pd.DataFrame(data[i]) if i != 'curriculum' else pd.DataFrame(pd.Series(data[i])).transpose() for i in data}
+    # dataframes present in dictionary `df_dict` (4) `content_item_orders`, `content_items`, `curriculum`, `curriculum_content_requirements`
+
+    for i in df_dict:
```

Comment:
If you do things in a loop like this then it's hard to make sure that stuff happens in a useful order. 

For example what happens if a ContentItemOrder is created before the ContentItems it should refer to are added to the database? 

Try this instead:

```
create_content_items(data['content_items'])  # write this as a seperate function
create_content_item_orders(.....)
create_curriculum(.....)
create_curriculum_content_requirements(.....)
```